components/map/layers/marker.md

```diff
@@ -84,144 +74,172 @@ The following example demonstrates how to configure the Marker layer of the Map.
 
     public class MarkerModel
     {
-        public double[] LatLng { get; set; }
-        public string Title { get; set; }
+        public double[]? LatLng { get; set; }
+        public string Title { get; set; } = string.Empty;
     }
 }
 ````
 
-## Customizing the Marker Appearance
+## Customizing Marker Appearance
 
-To customize the marker appearance, set the string `Template` parameter in the corresponding `MapLayerMarkerSettings` inner tag of the marker.
+To customize the marker appearance, set the `Template` parameter in the `<MapLayerMarkerSettings>` child tag of the corresponding `MapLayer`.
 
-The general syntax of the Marker template is based on the [Kendo UI templates](https://docs.telerik.com/kendo-ui/framework/templates/overview).
+The `Template` parameter must point to the name of a JavaScript function, which is defined in the global scope. This function can must return HTML markup as a string. The template function argument is a JavaScript object with properties that match the Marker layer's model class.
```

Comment:
There's a `can must` in here that needs to be resolved. Choose the best wording for accuracy.